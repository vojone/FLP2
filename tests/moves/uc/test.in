123
456
789
111 222 333 444
111 222 333 444
111 222 333 444
666
666
666
