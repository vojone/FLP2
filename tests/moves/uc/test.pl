:- consult("../../../cube").

:- read_cube(C, _), move_u_ccw(C, _, _, RC), write_cube(RC).
