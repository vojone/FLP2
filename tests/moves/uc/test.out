369
258
147
444 111 222 333
111 222 333 444
111 222 333 444
666
666
666
